src/components/Offers.jsx

```diff
@@ -37,31 +37,47 @@ const Offers = ({ request }) => {
   };
 
   const myOffers = request.offers.map((offer, index) => (
-    <OfferList offer={offer} index={index} onHelperClick={onHelperClick} />
+    <OfferList
+      offer={offer}
+      requestStatus={request.status}
+      index={index}
+      onHelperClick={onHelperClick}
+    />
   ));
 
+  const acceptedHelperOffer = request.offers.filter(
+    (offer) => offer.status === "accepted"
+  )[0];
+
   return (
-    <div style={{ marginLeft: "30px", display: "flex", flexDirection: "row" }}>
-      <List divided relaxed id="offers">
-        <h3>Offers</h3>
-        {myOffers}
-      </List>
-      <div
-        style={{
-          marginLeft: "30px",
-          display: "flex",
-          flexDirection: "column",
-          justifyContent: "space-between",
-        }}
-      >
-        {showHelperMessage && (
-          <OfferMessage
-            helperOffer={helperOffer}
-            onClickActivity={onClickActivity}
-          />
-        )}
-        <p id="status-message">{statusMessage}</p>
-      </div>
+    <div style={{ display: "flex", flexDirection: "row" }}>
+      {selectedStatus === "pending" && (
```

Comment:
Won't this act really weird if you are looking at a request, say from the active category, and then, without changing the request you are looking at, just change the status you are looking at?

Reply:
It is working well, but we still have an selectedRequest in redux, that could be reset on menu click